CHANGELOG.md

```diff
@@ -14,6 +14,8 @@ and what APIs have changed, if applicable.
 
 ## [Unreleased]
 
+## [29.69.2] - 2025-05-20
+- Skipping ZK node creation for Raw D2 Client for post commit
```

Comment:
hmm, there should be an empty line between here. Did you use the /scripts/update-changelog to update this file?